lib/search-bar/index.jsx

```diff
@@ -41,10 +41,11 @@ const mapStateToProps = ({ appState: state }) => ({
   showTrash: state.showTrash,
 });
 
-const mapDispatchToProps = (dispatch, { noteBucket }) => ({
+const mapDispatchToProps = (dispatch, { noteBucket, onNoteOpened }) => ({
   onNewNote: content => {
     dispatch(search({ filter: '' }));
     dispatch(newNote({ noteBucket, content }));
+    onNoteOpened();
```

Comment:
huh. would it be worth creating a new semantic action for this? `onNoteOpened()` leaves me flabbergasted when trying to understand what it's supposed to be doing. on the other hand, some action like `openNote()`, better `openNoteView()`, or something else that makes sense could clear it up some. is it worth the extra steps intros PR? not sure.

of note, `onNoteOpened` updates state to `isNoteOpen: true` which ends up being passed around and then sets a CSS class name - nothing else.

seems like we can go back to some of the naming troubles at the time and realize we lack a few simple terms to describe the views.

further, is there any time we would want to create a new note without opening up the note editor view? we've passed this prop down through the UI hierarchy but we're already calling `newNote()` in `app-state` - we could also listen for that action in `app-state` and _also_ update the current view.

for previous work see https://github.com/Automattic/simplenote-electron/pull/443/files#diff-ad1834ceae17437ce4883908cceeb573

Reply:
My concern is that doing so will take us down a path that will require an large effort and significant refactor that is not within the scope of this sprint.  

I used the existing method for opening the note:  https://github.com/Automattic/simplenote-electron/blob/develop/lib/note-list/index.jsx#L172

I would like to clean up the redux and flux state but should we stall this fix till that can be completed?

Reply:
sure. that's fine as long as we acknowledge it. it caught my eye because of the duplicated logic and the hard time I have knowing what "onNoteOpened" _should_ mean.

👍

Reply:
100% agree